XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -884,6 +890,10 @@
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
+    <Page Include="ControlPages\InkToolbarPage.xaml">
```

Comment:
This change isn't expected, right?

Reply:
I referred to the .csproj file in the master branch and saw that this was included, so I thought I should keep it included here as well.

Reply:
The trouble with the change is that it looks like it's adding another entry into the Pages table (meaning we have both this entry AND the one on line [745](https://github.com/microsoft/Xaml-Controls-Gallery/blob/acfb732c96a18c30258d53bc7324af0b8fd4ad2c/XamlControlsGallery/XamlControlsGallery.csproj#L745)). My guess is that this won't build.

Reply:
Gosh sorry, not sure why all of this got so jumbled. You're right. I will fix that and push

Reply:
Thanks for cleaning this up!